src/main/java/edu/wpi/first/outlineviewer/OutlineViewer.java

```diff
@@ -15,9 +16,24 @@
 
   private static final ButtonType START = new ButtonType("Start", ButtonData.OK_DONE);
   private static final ButtonType QUIT = new ButtonType("Quit", ButtonData.CANCEL_CLOSE);
+  private static final String REQUIRED_JRE_VERSION = "1.8";
+  private static final String JRE_VERSION = System.getProperty("java.specification.version");
 
   @Override
   public void start(Stage primaryStage) throws IOException {
+    if (!REQUIRED_JRE_VERSION.equals(JRE_VERSION)) {
+      Alert invalidVersionAlert = new Alert(Alert.AlertType.ERROR);
+      invalidVersionAlert.setHeaderText("Invalid JRE Version!");
+      invalidVersionAlert.setContentText(
+          String.format("You are using an unsupported Java version: %s!  "
+              + "Please download Java %s.",
```

Comment:
This is going to say "Java 1.8", correct?  It's commonly known as Java 8, I don't know if Oracle calls it 1.8 anywhere.

Reply:
I'll hardcode it to say 8.